modules/backend/widgets/form/assets/js/october.form.js

```diff
@@ -220,6 +221,54 @@
             .nextUntil('.section-field').hide()
     }
 
+    /*
+     * Bind validation checks on available fields, triggered when
+     * the fields is blurred or changed
+     */
+    FormWidget.prototype.bindValidation = function () {
+        let fields = this.getFieldElements(),
+            form = this.$form;
+
+        for (let field of fields) {
```

Comment:
I think October is still supposed to support at the very least IE 11?

Reply:
I wasn't sure on which browsers October was still targeting, as I couldn't find anything in the docs. I'll keep in mind that IE11 is still needing to be supported.